plugins/openshift-developer/skills/address-review-pr/scripts/check_authorized.py

```
@@ -0,0 +1,234 @@
+    except Exception as e:
+        errors.append(f"OWNERS: {e}")
+
+    if errors:
+        raise OwnersLookupError("; ".join(errors))
+
+    return authorized
+
+
+def is_org_member(owner: str, login: str) -> bool:
+    try:
+        result = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)